src/buildDb.ts

```diff
@@ -0,0 +1,15 @@
+import { allLaptops } from './LaptopDatabase';

```

Comment:
If this isn't a site source I'd probably discourage putting it here 🤔. Maybe make a scripts folder and move this there to keep things clean?

Reply:
That's a good idea. Is there anything I need to do to make the linter still work with it?

Reply:
Since the linter points to the TS config, I'm guessing I just have to do something like this? https://stackoverflow.com/a/64286953

Reply:
Actually, I just realized I probably just have to add the scripts directory to the `npm run lint` command definition. Let me give that a shot.

Reply:
Okay, that seems to have worked! If you're good with it, resolve this conversation and I'll merge it.